### PR TITLE
add mdx to prettier extensions

### PR DESCRIPTION
**Pull request by gwyneplaine**
`yarn format` wasn't picking up mdx files in our repo. (which we now have an abundance of)

### CODE REVIEW

**Review by JedWatson — APPROVED** [review]

File: docs-next/pages/apis/system.mdx [hunk 1]
```diff
@@ -144,7 +144,7 @@ Options:
 
 - `isAccessAllowed` (default: `(context) => !!context.session`): This function controls whether a user is able to access the Admin UI.
   It takes a [`KeystoneContext`](./context) object as an argument.
-- `path` (default: `'admin'` ): The URL path where your Admin UI will be accessible from.
+- `path` (default: `'admin'` ): The URL path where you Admin UI will be accessible from.
```
Comment (mitchellhamilton) [inline docs-next/pages/apis/system.mdx:147]: Was this change intentional?

File: packages/file-adapters/README.md [hunk 2]
```diff
@@ -153,14 +153,14 @@ const fileAdapter = new S3Adapter({
...
-| `publicUrl`    | `Function`             |             | By default the publicUrl returns a url for the S3 bucket in the form `https://{bucket}.s3.amazonaws.com/{key}/{filename}`. This will only work if the bucket is configured to allow public access.                                       |
-| `s3Options`    | `Object`               | `undefined` | For available options refer to the [AWS S3 API](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html)                                                                                                                         |
-| `uploadParams` | `Object    | Function` | `{}`        | A config object or function returning a config object to be passed with each call to S3.upload. For available options refer to the [AWS S3 upload API](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property). |
+| Option         | Type       | Default     | Description                                                                                                                                                                                        |
```
Comment (mitchellhamilton) [inline packages/file-adapters/README.md:156]: This table seems to be broken(at least on GH)

Comment (gwyneplaine) [reply packages/file-adapters/README.md:156]: yeah, I'm reverting the change, seems like prettier is still doing funky things with md

File: docs-next/pages/apis/system.mdx (hunk 1 above)
Comment (gwyneplaine) [reply docs-next/pages/apis/system.mdx:147]: no, needs a rebase from main

**Comment by gwyneplaine** [comment]
Added a JiRA task to look more deeply into removing remark later
